format

```diff
@@ -40,7 +40,6 @@ find . -name '*.nix' \
   ! -path ./modules/services/sxhkd.nix \
   ! -path ./modules/systemd.nix \
   ! -path ./nix-darwin/default.nix \
-  ! -path ./tests/default.nix \
```

Comment:
Would it be possible to move this formatting change to its own PR? If it's a hassle then no worries about keeping it here.